build.gradle

```diff
@@ -22,15 +22,15 @@ buildscript {
 }
 
 repositories {
+    google()
     jcenter()
     mavenLocal()
 }
 
 dependencies {
     compile gradleApi()
     compile localGroovy()
-
-    compile 'com.android.tools.build:gradle:2.3.0'
+    compile 'com.android.tools.build:gradle:3.0.1'
```

Comment:
A little bit concerned about this. I'm curious to see how many projects are still using 2.3.0.

Reply:
I don't think it's a problem - if a project wants to use the new version of this library, they should upgrade their gradle version too.

Reply:
While we are at this, we should add a changelog